lib/rig_cloud_events/cloud_event.ex

```diff
@@ -85,22 +93,40 @@ defmodule RigCloudEvents.CloudEvent do
 
   # ---
 
-  defp specversion_0_1?(parsed) do
+  defp specversion_0_1?(parsed) when is_list(parsed) do
```

Comment:
this is .... weird. but only temporary anyway, I assume

Reply:
Yes, temporary, we should do another PRs to migrate also other stuff to use the couldevents lib.